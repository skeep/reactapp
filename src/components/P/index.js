// @flow

import React from 'react';

type Props = {
  content: string
};

const P = (props: Props): any => (<p>{props.content}</p>);

P.defaultProps = {
  content: "This is default content of a paragraph tag"
};

export default P;
