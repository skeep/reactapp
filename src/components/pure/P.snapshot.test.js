import React from 'react';
import P from './index.js';
import renderer from 'react-test-renderer';

it('renders correctly content 1', () => {
  const tree = renderer.create(
    <P content="content 1"/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly content 2', () => {
  const tree = renderer.create(
    <P content="content 2"/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
