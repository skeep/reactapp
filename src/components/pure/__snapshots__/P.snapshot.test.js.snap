// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly content 1 1`] = `
<p>
  content 1
</p>
`;

exports[`renders correctly content 2 1`] = `
<p>
  content 2
</p>
`;
